src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -142,6 +142,31 @@ OutputVector translate_mul_op(const NodeContext& node) {
     set_node_name(node.get_name(), result);
     return {result};
 }
+
+OutputVector translate_addv2_op(const NodeContext& node) {
+    default_op_checks(node, 2, {}, true);
+    auto lhs = node.get_input(0);
+    auto rhs = node.get_input(1);
+    auto result = make_shared<v1::Add>(lhs, rhs);
```

Comment:
Please move this line below `if`. That is because you may add two outputs from `ComplexTypeMark` here.